From 15e5ebb116ffa0a1df808369f9447eda13512dde Mon Sep 17 00:00:00 2001
From: Sam Hartman <hartmans@debian.org>
Date: Thu, 18 Sep 2014 15:55:50 -0400
Subject: eap.server.key.diff

---
 raddb/eap.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/raddb/eap.conf b/raddb/eap.conf
index ac93fd2..3ae75fb 100644
--- a/raddb/eap.conf
+++ b/raddb/eap.conf
@@ -156,7 +156,7 @@
 			cadir = ${confdir}/certs
 
 			private_key_password = whatever
-			private_key_file = ${certdir}/server.pem
+			private_key_file = ${certdir}/server.key
 
 			#  If Private key & Certificate are located in
 			#  the same file, then private_key_file &
